examples/posenet/src/helperFunc.js

```diff
@@ -28,7 +30,8 @@ function getInput(inputElement) {
 
 function getURL(version) {
   let address;
-  const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  // const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  const urlBase = '../posenet/model/';
```

Comment:
are we going to host the model files ourselves?

Reply:
Will continue to use https://storage.googleapis.com/tfjs-models/weights/posenet/ by default.

Reply:
Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac